src/Confluent.Kafka/Consumer.cs

```diff
@@ -367,7 +367,7 @@ private void ErrorCallback(IntPtr rk, ErrorCode err, string reason, IntPtr opaqu
         {
             // TODO: Is reason ever different from that returned by err2str?
```

Comment:
I missed the same spot in the Producer. Would be good if code like this was in a shared place instead. DRY and all that